yt_dlp/extractor/vrt.py

```diff
@@ -1,82 +1,215 @@
-        'md5': 'e1663accf5cf13f375f3cd0d10476669',
-        'info_dict': {
-            'id': 'pbs-pub-7855fc7b-1448-49bc-b073-316cb60caa71$vid-2ca50305-c38a-4762-9890-65cbd098b7bd',
-            'ext': 'mp4',
-            'title': 'Beelden van binnenkant Notre-Dame, één maand na de brand',
-            'description': 'Op maandagavond 15 april ging een deel van het dakgebinte van de Parijse kathedraal in vlammen op.',
-            'timestamp': 1557924660,
-            'upload_date': '20190515',
-            'duration': 31.2,
-        },
-    }, {
-        'url': 'https://sporza.be/nl/2019/05/15/de-belgian-cats-zijn-klaar-voor-het-ek/',
-        'md5': '910bba927566e9ab992278f647eb4b75',
+        'url': 'https://www.vrt.be/vrtnu/a-z/de-ideale-wereld/2023-vj/de-ideale-wereld-d20230116/',
         'info_dict': {
-            'id': 'pbs-pub-f2c86a46-8138-413a-a4b9-a0015a16ce2c$vid-1f112b31-e58e-4379-908d-aca6d80f8818',
+            'id': 'pbs-pub-855b00a8-6ce2-4032-ac4f-1fcf3ae78524$vid-d2243aa1-ec46-4e34-a55b-92568459906f',
+            'title': 'Tom Waes',
+            'thumbnail': 'https://images.vrt.be/orig/2023/01/10/1bb39cb3-9115-11ed-b07d-02b7b76bf47f.jpg',
             'ext': 'mp4',
-            'title': 'De Belgian Cats zijn klaar voor het EK mét Ann Wauters',
-            'timestamp': 1557923760,
-            'upload_date': '20190515',
-            'duration': 115.17,
+            'duration': 1939.0,
+            'release_date': '20230116',
+            'description': 'Satirisch actualiteitenmagazine met Ella Leyers. Tom Waes is te gast.',
+            'display_id': 'de-ideale-wereld-d20230116',
+            'episode_number': 1,
+            'timestamp': 1673905125,
+            'series': 'De ideale wereld',
+            'upload_date': '20230116',
+            'channel': 'VRT',
+            'season_id': '1672830988794',
+            'episode_id': '1672830988861',
+            'release_timestamp': 1673905125,
+            'episode': 'Aflevering 1'
         },
-    }, {
-        'url': 'https://www.vrt.be/vrtnws/en/2019/05/15/belgium_s-eurovision-entry-falls-at-the-first-hurdle/',
-        'only_matching': True,
-    }, {
-        'url': 'https://www.vrt.be/vrtnws/de/2019/05/15/aus-fuer-eliott-im-halbfinale-des-eurosongfestivals/',
-        'only_matching': True,
     }]
-    _CLIENT_MAP = {
-        'vrt.be/vrtnws': 'vrtnieuws',
-        'sporza.be': 'sporza',
+    _NETRC_MACHINE = 'vrtnu'
+    _APIKEY = '3_0Z2HujMtiWq_pkAjgnS2Md2E11a1AwZjYiBETtwNE-EoEHDINgtnvcAOpNgmrVGy'
+    _CONTEXT_ID = 'R3595707040'
+    _REST_API_BASE_TOKEN = 'https://media-services-public.vrt.be/vualto-video-aggregator-web/rest/external/v2'
+    _REST_API_BASE_VIDEO = 'https://media-services-public.vrt.be/media-aggregator/v2'
+    _HLS_ENTRY_PROTOCOLS_MAP = {
+        'HLS': 'm3u8_native',
+        'HLS_AES': 'm3u8_native',
     }
 
+    _authenticated = False
```

Comment:
we can inline all of this except `_authenticated` imo
```suggestion
    _authenticated = False
```